packages/api/src/services/actions.ts

```diff
@@ -79,6 +79,17 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       // convert this to tuple to save bytes.
       return results.map(({ price, timestamp }) => [timestamp, price]);
     },
+    async getEmpStats(address: string, currency: CurrencySymbol = "usd") {
+      assert(address, "requires address");
+      assert(currency, "requires currency");
```

Comment:
I'm guessing these checks could never fail since `"usd"` is assigned above, but they're left in the code in case the default assignment is removed?

Reply:
thats a good point, its possible for the frontend caller to override the default by passing null explicitly ( or any falsy value), its an edge case but this would catch it